standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -555,6 +555,9 @@ public static ConfVars getMetaConf(String name) {
     EVENT_DB_LISTENER_TTL("metastore.event.db.listener.timetolive",
         "hive.metastore.event.db.listener.timetolive", 86400, TimeUnit.SECONDS,
         "time after which events will be removed from the database listener queue"),
+    CLEAN_MAX_EVENTS("metastore.event.db.clean.maxevents",
+            "hive.metastore.event.db.clean.maxevents", 10000,
+            "Limit on number events to be cleaned at a time in metastore cleanNotificationEvents call, to avoid OOM"),
```

Comment:
Also, shall mention that if passed as 0 or negative values, then all expired events will be cleaned.

Reply:
Done.